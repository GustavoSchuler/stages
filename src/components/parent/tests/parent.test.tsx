import React from "react"
import { render, cleanup, waitFor } from '@testing-library/react';
import '@testing-library/jest-dom';

import Parent from '../parent.component';

describe('[Component] Parent', () => {
  let wrapper: any;

  beforeEach(() => {
    jest.useFakeTimers();

    wrapper = render(
      <Parent />
    );
  });

  afterEach(cleanup);

  it('renders correctly', () => {
    const element = wrapper.getByTestId('parent');

    expect(element).toMatchSnapshot();
  });
});
