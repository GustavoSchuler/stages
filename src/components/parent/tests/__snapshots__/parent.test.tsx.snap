// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component] Parent renders correctly 1`] = `
<div
  class="sc-fzoLsD CSqIi"
  data-testid="parent"
>
  <h1
    class="sc-fzpjYC bGlwZc"
  >
    Stages
  </h1>
  <div
    class="sc-AxjAm jVdwCl"
    data-testid="child__box"
    display="flex"
    height="128"
  >
    <div
      class="sc-fzoXWK fcyArp"
      color="#FFEB3B"
      data-testid="child__loader"
      opacity="0.7"
      size="48"
      speed="2"
    >
      <div />
      <div />
    </div>
  </div>
</div>
`;
